src/FormParserFactory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace React\Http;
+
+use React\Http\Parser;
+
+class FormParserFactory
```

Comment:
What does this "parse"? Not sure, how about a `FormParser` or `BodyParser` or `PayloadParser` namespace?

Then again, what does this class do?